gammapy/utils/tests/test_time.py

```diff
@@ -0,0 +1,14 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from ..time import time_ref_from_dict
+
+def test_time_ref_from_dict():
+    mjd_int = 500
+    mjd_frac = 0.5
+    time_ref_dict = dict(MJDREFI=mjd_int, MJDREFF=mjd_frac)
+    time_ref = time_ref_from_dict(time_ref_dict)
+    decimal = 4
+    s_error = "time reference not compatible with defined values"
```

Comment:
Please don't add error messages to tests. It adds a cost (work to write / review them and read them when you read a test) and almost no benefit (tests that pass once very rarely fail, and if they do the traceback and looking at the code is what's useful, not some printed message).